### ISSUE TITLE
Question about using v5

### ISSUE THREAD

**Issue by Duncan00** [ajv-validator/ajv#204]
wanna to try out `v5`, but got this message after setting `v5: true`

Error: no schema with key or ref "https://raw.githubusercontent.com/epoberezkin/ajv/master/lib/refs/json-schema-v5.json"

Thanks

**Comment by epoberezkin** [comment]
Could you post the code please?

**Comment by Duncan00** [comment]
I am only using `v4` schemas, and only add `v5: true` config.

``` javascript
Ajv({
    v5: true,

    allErrors: true,
    removeAdditional: false,
    verbose: true,
    format: 'full',
    formats: {},
    schemas: _.values(schemas),
    useDefaults: true,
    meta: true,
    validateSchema: true,
    inlineRefs: true,
    missingRefs: true,
    uniqueItems: true,
    unicode: true,
    beautify: false,
    jsonPointers: false,
    messages: true
})
```

Thanks~

**Comment by epoberezkin** [comment]
https://tonicdev.com/esp/574e756875fd9d13001bdbd7

**Comment by epoberezkin** [comment]
The issue is in some of your schemas probably...

**Comment by epoberezkin** [comment]
What is the version of ajv?

**Comment by epoberezkin** [comment]
Ok, I think I see the issue. As a workaround, instead of the option `schemas: _.values(schemas)` use `ajv.addSchemas(_.values(schemas))` for now. Will fix

**Comment by Duncan00** [comment]
it is `4.1.0`~ 
i will try~ 
Thanks

**Comment by epoberezkin** [comment]
Or better add `$schema` keyword in all your schemas. The bug is that with v5 option it tries to use v5 metaschema by default but at the time `schemas` option is processed it is not added yet. If you tell your schemas to use draft4 metaschema explicitly it will also work.

**Comment by epoberezkin** [comment]
fixed in 4.1.1
